packages/website/src/features/Docs/DocsCannonfilesPage.tsx

```diff
@@ -411,7 +419,10 @@ export const DocsCannonfilesPage: FC = () => {
                     #
                   </Link>
                 </Heading>
-                <Text color="gray.400">Coming soon.</Text>
+                <Text color="gray.400">
+                  Contains artifact data and other contract source code data
+                  about the contracts deployed during the build.
```

Comment:
We really need to expand on this with detailed information of what is *inside* the package code. Would be good to show the contents/explanation of the fields inside the data, and how to pull/parse it out.